src/main/java/org/spongepowered/api/resourcepack/ResourcePackFactory.java

```diff
@@ -28,7 +28,7 @@
 import java.net.URL;
 
 /**
- * A factory for creating {@link ResourcePack}s.
+ * A provider for creating {@link ResourcePack}s.
```

Comment:
I have a feeling your find & replace borked this, along with other event-related classes.